recipes.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Recipes</title>
+</head>
+<body>
+  <p>
+    <h2>Recipes</h2> ...and tasty bites from <em>all around the world.</em>
+  </p>
+
+  <div>
+    <ul>
+      <li>
+        <a href= "http://www.seriouseats.com/recipes/2015/03/the-best-japanese-pork-and-cabbage-dumplings-gyoza-recipe.html">Japanese Pork and Cabbage Dumplings</a>
```

Comment:
these a tags are all pretty long, having their visible content be on a separate line  from the href or opening tag might make editing later easier.